src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -4523,7 +4531,7 @@ private SymbolInfo GetNamedArgumentSymbolInfo(IdentifierNameSyntax identifierNam
             }
             else
             {
-                var symbols = ArrayBuilder<ISymbol>.GetInstance();
```

Comment:
>var symbols = ArrayBuilder<ISymbol>.GetInstance(); [](http://example.com/codeflow?start=16&length=50)

Do we have an evidence that this code path has the same problem as ```LookupSymbolsInternal```? I can see how ```LookupSymbolsInternal``` could have thousands of items in the result when ```name``` is not specified. In this case, however, it looks like we are dealing with candidate symbols for a specific invocation/property access, there aren't usually that many of them.

Reply:
Same as before, this method didn't show up in the profile, but was trying to maintain consistency. Very willing to change back if preferred.

Reply:
> Same as before, this method didn't show up in the profile, but was trying to maintain consistency. 

Same response here as well

Reply:
Reverted this one